File: WordApp/Models/TaskModels/SentenceTaskModels/AnsweredSentenceModel.cs
using WordApp.Models.Base;
using WordApp.Models.Sentences;

namespace WordApp.Models.TaskModels.SentenceTaskModels
{
    public class AnsweredSentenceModel: BaseModel
    {
        public SentenceAnswerModel Answer { get; set; }
        public SentenceModel Sentence { get; set; }
    }
}

File: WordApp/ClientApp/src/app/models/tasks/sentences/answered-sentence.model.ts
import { SentenceModel } from './../../sentences/sentence.model';
import { TaskAnswerModel } from './../../../tasks/models/task-answer.model';

export class AnsweredSentenceModel {
  sentence: SentenceModel;
  answer: TaskAnswerModel;

  constructor(sentence: SentenceModel, answer: TaskAnswerModel) {
    this.sentence = sentence;
    this.answer = answer;
  }
}
